### ISSUE TITLE
Workflow marking_store arguments are not compatible with Symfony 5.x anymore

### ISSUE THREAD

**Issue by mugge6** [pimcore/pimcore#9525]
# Bug Report

### Expected behavior

Symfony 4.3 configuration:
```
   marking_store:
      type: single_state
      arguments:
         - workflowState
```

Symfony 4.4 and 5.x configuration:
```
            marking_store:
                type: 'method'
                property: 'currentPlace'
```

Since Pimcore X property value is always default "marking" and is not configurable with "arguments" setting from Pimcore over configuration.

**Comment by brusch** [comment]
See also https://github.com/symfony/symfony/blob/5.4/UPGRADE-5.0.md#workflow

**Comment by mugge6** [comment]
This issue should be fixed with high priority. If someone is using workflows and defined the markingstore property to another value than the default value "marking", then its not even possible to save a DataObject. At least if workflows are used on this type.

**Comment by brusch** [comment]
@mugge6 providing a PR would speed things up a lot 😉

**Comment by dpfaffenbauer** [comment]
I fixed it yesterday, but did a composer update and reverted all the changes again... shouldn't be too difficult, since you can basically merge from Symfony FrameworkExtension and keep the extensions Pimcore adds.

**Comment by kyoun8** [comment]
Hi, any news regarding this bugfix ?

**Comment by brusch** [comment]
@kyoun8 see my comment above - a pull request would help 😉

**Comment by pimcore-galilee** [comment]
Hello all. Any news on this topic?

**Comment by brusch** [comment]
Fixed by #10092
